gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -166,6 +166,17 @@ def get_relatedness_annotated_ht() -> hl.Table:
     "gs://gcp-public-data--gnomad/resources/grch38/gnomad_v4_qc_sites_b38.ht"
 )
 
+# gnomAD v4 dense MT of all predetermined possible QC sites `predetermined_qc_sites`
+v4_predetermined_qc = VersionedMatrixTableResource(
```

Comment:
@klaricch I accidentally deleted your comment
"could maybe add more details in this comment how these were predetermined" 

This was my response that I also accidentally deleted: Actually, this will be resolved when my other PR get's merged, this file is also added there, so I will merge with main after that one goes in